packages/blocks/src/__internal__/utils/types.ts

```diff
@@ -247,6 +247,8 @@ export type SerializedBlock = {
   children: SerializedBlock[];
   sourceId?: string;
   caption?: string;
+  name?: string;
```

Comment:
Are we going to deprecate using this type? /cc @Saul-Mirone @fourdim

Reply:
Ok. Because the attachment block has the name property.

Reply:
> Are we going to deprecate using this type? /cc @Saul-Mirone @fourdim

Yep